src/textual_summary/generic_table_row.jsx

```diff
@@ -47,7 +47,9 @@ GenericTableRow.propTypes = {
     title: PropTypes.string,
     image: PropTypes.string,
     icon: PropTypes.string,
-    value: PropTypes.any,
+    label: PropTypes.any,
```

Comment:
We should avoid using `PropTypes.any`. I know that sometimes it can be difficult, especially dorm some dynamic objects, but props like `label` should be defined properly.

Also, if you don't mark object attributes as required, prop validation will pass even for empty object. If there are some conditions, when should be attribute required, and whet it should be undefined, you can create custom prop validator. You can  check this example: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Right. I did not implement describe all the data types in a lot of detail. I plan to improve that in follow up PRs. Together with the use of patternfly-react components where applicable.